slack_gpt_bot.py

```diff
@@ -118,16 +120,23 @@ def command_handler(body, context):
             channel_id=channel_id, 
             user=user.username, 
             email=user.email,
-            request=messages[1:],   #field 0 is something that slack adds that we don't need
+            request=messages[1:],   #field 0 is something that gets added as part of process_conversation_history that we don't need
             response=response_text
         )
     
     except Exception as e:
-        logging_wrapper("Exception", logging.ERROR, exception=e)
+        logging_wrapper("Exception", logging.ERROR, 
+            token_count=num_tokens,
+            channel_id=channel_id, 
+            user=user.username, 
+            email=user.email,
+            request=messages[1:],
+            exception=e
+        )
```

Comment:
I like having the exceptions log the same user details as the successful calls. 👍